src/sentry/incidents/action_handlers.py

```diff
@@ -24,8 +29,96 @@ def resolve(self):
 
 @AlertRuleTriggerAction.register_type_handler(AlertRuleTriggerAction.Type.EMAIL)
 class EmailActionHandler(ActionHandler):
+    query_aggregations_display = {
+        QueryAggregations.TOTAL: "Total Events",
+        QueryAggregations.UNIQUE_USERS: "Total Unique Users",
```

Comment:
Down the road, will this be open to using any custom aggregation or aggregation alias?

Reply:
We haven't planned for that at all with incidents/alert rules. It's not impossible though.